ast.go

```diff
@@ -1977,3 +2065,85 @@ func (node *Revoke) walkSubtree(visit Visit) error {
 	}
 	return Walk(visit, node.Privileges, node.Table)
 }
+
+// resolvers
+
+func resolveReadStatementWalk(node Node, resolver ReadStatementResolver) (string, error) {
+	err := Walk(func(node Node) (bool, error) {
+		if funcExpr, ok := node.(*CustomFuncExpr); ok && funcExpr != nil {
+			resolvedString, err := resolveReadStatement(funcExpr, resolver)
+			if err != nil {
+				return true, err
+			}
+			funcExpr.ResolvedString = resolvedString
+		}
+		return false, nil
+	}, node)
+	if err != nil {
+		return "", err
+	}
+	return node.String(), nil
+}
+
+func resolveReadStatement(node *CustomFuncExpr, resolver ReadStatementResolver) (string, error) {
+	switch node.Name {
+	case "block_num":
```

Comment:
Q: Is `node.Name` always normalized to be lowercase?

Reply:
In this case, it will be because [they were set in lowercase](https://github.com/tablelandnetwork/go-sqlparser/pull/42/files#diff-a6ec253416e8e9e6846f73f5c7677d6ff6fa85f5c37317f84e608fc50ff8238eR984-R999)